modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/service/impl/BatchEngineExportTaskServiceImpl.java

```diff
@@ -30,6 +30,25 @@
 public class BatchEngineExportTaskServiceImpl
 	extends BatchEngineExportTaskServiceBaseImpl {
 
+	@Override
+	public BatchEngineExportTask getBatchEngineExportTask(Long exportTaskId)
+		throws PortalException {
+
+		BatchEngineExportTask entry =
```

Comment:
To fit the pattern, I think `entry` should be renamed to `batchEngineExportTask`